pifi/videoprocessor.py

```diff
@@ -386,28 +412,7 @@ def __get_mbuffer_cmd(self, buffer_size_bytes, log_file = None):
             log_file_clause = f' -l {log_file} '
         return f'mbuffer -q {log_file_clause} -m ' + shlex.quote(str(buffer_size_bytes)) + 'b'
 
-    # Fps is available in self.__video_info metadata obtained via youtube-dl, but it is less accurate than using ffprobe.
```

Comment:
would you mind preserving this comment? Perhaps you can put it where you moved the `ffprobe` invocation.